src/main/java/aquality/selenium/browser/BrowserTabNavigation.java

```diff
@@ -0,0 +1,90 @@
+package aquality.selenium.browser;
+
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import static java.lang.String.format;
+
+class BrowserTabNavigation implements IBrowserTabNavigation {
+
+    private final RemoteWebDriver driver;
+
+    private void infoLoc(String key, Object... args) {
+        AqualityServices.getLocalizedLogger().info(key, args);
+    }
```

Comment:
Please move all methods below the constructor

Reply:
fixed